recipes-soletta/soletta/soletta_git.bb

```diff
@@ -105,6 +105,9 @@ do_configure_append() {
        sed -i -e 's/^ *FLOW_NODE_TYPE_GTK=.*/# FLOW_NODE_TYPE_GTK is not set/g' ${S}/.config
    fi
 
+   sed -i "s/^\(CONFIG_LDFLAGS=\).*/\1\"${TARGET_LDFLAGS}\"/g" ${S}/.config
```

Comment:
```${S}/.config``` does not yet exist in ```do_configure_prepend()```. cml1 bbclass runs oldconfig after which ```${S}/.config``` becomes available for sed'ing.